crates/iroha_data_model/src/trigger.rs

```diff
@@ -249,6 +236,13 @@ pub mod action {
         }
     }
 
+    impl Repeats {
+        /// Returns `true` if this repeat policy has no remaining executions.
+        pub fn is_depleted(&self) -> bool {
+            matches!(self, Repeats::Exactly(0))
+        }
+    }
```

Comment:
Forgive me my pedantry, but this is a perfect candidate for a trivial and _useful_ unit test.